### PR TITLE
Added SELinux post-deployment information (jsc#PED-2662)

### PR DESCRIPTION
**Pull request by tbazant**
### Are there any relevant issues/feature requests?

Fixes https://jira.suse.com/browse/PED-2662

### CODE REVIEW

File: xml/task-alp-post-deployment-considerations.xml [hunk 1]
```diff
@@ -63,6 +63,45 @@
...
+<screen>&prompt.root;setenforce 0</screen>
+    <para>
+      To monitor AVCs, look for log messages in the Audit log and &systemd;
+      journal, for example:
+    </para>
+<screen>&prompt.root;tail -f /var/log/audit/audit.log | grep -i AVC</screen>
+    <para>
+      and
+    </para>
+<screen>&prompt.root;journalctl -f | grep -i AVC</screen>
```
Comment (jsegitz) [inline xml/task-alp-post-deployment-considerations.xml:94]: Please also recommend 
`ausearch -m avc,user_avc,selinux_err -i`
for more advanced searches

File: xml/task-alp-post-deployment-considerations.xml [hunk 2]
```diff
@@ -63,6 +63,45 @@
...
+    <para>
+      If such messages appear while using the application that did not behave
+      correctly when &selnx; was set to the enforce mode, the policies are
+      probably too restrictive and need updating. You can help to fine-tune
+      &selnx; policies by creating a bug report at
+      <link xlink:href="https://bugzilla.suse.com/enter_bug.cgi?classification=SUSE%20ALP%20-%20SUSE%20Adaptable%20Linux%20Platform"/>.
+      Specify <literal>Basesystem</literal> as a component and attach gathered
+      unique lines that include AVCs.
```
Comment (jsegitz) [inline xml/task-alp-post-deployment-considerations.xml:102]: Please include `SELinux` in the bug subject, we monitor for that

File: xml/task-alp-post-deployment-considerations.xml [hunk 3]
```diff
@@ -63,6 +63,45 @@
...
+      &alpshort; ships with &selnx; enabled and set to the restricting
+      <emphasis>enforce</emphasis> mode for increased security. The enforce
+      mode can lead to processes or workloads not behaving correctly because
+      the default policy may be too strict. If you observe such unexpected
+      issues, set &selnx; to the <emphasis>permissive</emphasis> mode that does
+      not enforce &selnx; policies but still gathers offences against them
+      called <emphasis>Access Vector Rules</emphasis> (AVCs).
+    </para>
+    <para>
+      To set &selnx; into the permissive mode, run:
```
Comment (jsegitz) [inline xml/task-alp-post-deployment-considerations.xml:83]: temporarily. to set it to permissive mode permanently edit `/etc/selinux/config`

**Review by jsegitz — CHANGES_REQUESTED** [review]

File: xml/task-alp-post-deployment-considerations.xml [hunk 4]
```diff
@@ -63,6 +63,79 @@
       </para>
     </section>
   </section>
+  <section xml:id="alp-post-deploy-selinux">
+    <title>&selnx;</title>
+    <para>
+      Security-Enhanced Linux (&selnx;) is a security framework that increases
+      system security by defining access controls for applications, processes,
+      and file on the file system.
```
Comment (dariavladykina) [inline xml/task-alp-post-deployment-considerations.xml:71]:
```suggestion
      system security by defining access controls for applications, processes
      and files on the file system.
```

File: xml/task-alp-post-deployment-considerations.xml [hunk 5]
```diff
@@ -63,6 +63,79 @@
...
+    </para>
+    <para>
+      &alpshort; ships with &selnx; enabled and set to the restricting
```
Comment (dariavladykina) [inline xml/task-alp-post-deployment-considerations.xml:74]:
```suggestion
      &alpshort; ships with &selnx; enabled and set to the restrictive
```

File: xml/task-alp-post-deployment-considerations.xml [hunk 6]
```diff
@@ -63,6 +63,79 @@
...
+      <emphasis>enforce</emphasis> mode for increased security. The enforce
+      mode can lead to processes or workloads not behaving correctly because
+      the default policy may be too strict. If you observe such unexpected
+      issues, set &selnx; to the <emphasis>permissive</emphasis> mode that does
+      not enforce &selnx; policies but still log offences against them called
```
Comment (dariavladykina) [inline xml/task-alp-post-deployment-considerations.xml:79]:
```suggestion
      not enforce &selnx; policies but still logs offences against them called
```

File: xml/task-alp-post-deployment-considerations.xml [hunk 7]
```diff
@@ -63,6 +63,79 @@
...
+      <para>
+        To set &selnx; to the permissive mode permanently, edit
+        <filename>/etc/selinux/config</filename> and update it to include the
+        following line:
+      </para>
+<screen>SELINUX=permissive</screen>
+    </tip>
+    <important>
+      <para>
+        If you entered a &selnx; permissive mode, you need to relabel your
```
Comment (dariavladykina) [inline xml/task-alp-post-deployment-considerations.xml:96]:
```suggestion
        If you entered an &selnx; permissive mode, you need to relabel your
```

File: xml/task-alp-post-deployment-considerations.xml [hunk 8]
```diff
@@ -63,6 +63,79 @@
...
+<screen>&prompt.root;ausearch -m avc,user_avc,selinux_err -i</screen>
+    <para>
+      If such messages appear while using the application that did not behave
+      correctly when &selnx; was set to the enforce mode, the policies are
+      probably too restrictive and need updating. You can help to fine-tune
+      &selnx; policies by creating a bug report at
+      <link xlink:href="https://bugzilla.suse.com/enter_bug.cgi?classification=SUSE%20ALP%20-%20SUSE%20Adaptable%20Linux%20Platform"/>.
+      Specify <literal>Basesystem</literal> as a component, include the word
+      <literal>&selnx;</literal> in the bug subject, and attach gathered unique
+      lines that include AVCs together with reproduction steps.
```
Comment (dariavladykina) [inline xml/task-alp-post-deployment-considerations.xml:136]:
```suggestion
      <literal>&selnx;</literal> in the bug subject, and attach the gathered
      unique lines that include AVCs together with reproduction steps.
```

**Review by dariavladykina — COMMENTED** [review]
Please consider some suggestions, thanks!

**Pull request merged**